assets/scss/components/_bounds.scss

```diff
@@ -13,12 +13,12 @@ padding, and sets a max-width.
 	box-sizing: border-box;
 	margin-left: auto;
 	margin-right: auto;
-	max-width: $bounds-wide;
+	max-width: $bounds;
```

Comment:
i thought we wanted bounds wide, and then perhaps there is a prop add to bounds narrow